nvflare/widgets/fed_event.py

```diff
@@ -104,6 +102,11 @@ def _receive(self, topic: str, request: Shareable, fl_ctx: FLContext) -> Shareab
             return make_reply(ReturnCode.BAD_REQUEST_DATA)
 
         with self.in_lock:
+            if self.poster is None:
+                # create the poster thread now
+                self.poster = threading.Thread(target=self._post, args=())
+                self.poster.start()
```

Comment:
should we change the line in `__init__.py` to `self.poster = None`

Reply:
Changed as suggested.